src/brod_sock.erl

```diff
@@ -245,7 +246,16 @@ maybe_sasl_auth(Sock, Mod, ClientId, Timeout,
           exit({sasl_auth_error, Unexpected})
       end;
     _ -> exit({sasl_auth_error, ErrorCode})
-  end.
+  end;
+maybe_sasl_auth(Host, Sock, Mod, ClientId, Timeout,
+    _SaslOpts = {callback, ModuleName, Opts}) ->
+    case brod_auth_backend:auth(ModuleName, Host, Sock, Mod,
+        ClientId, Timeout, Opts) of
```

Comment:
nit: please align to the column next to `(` the line above.
plus 225 in this module
and lines in brod_auth_backend.erl